### ISSUE TITLE
Mini FAB needs hit target of at least 48x48

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#16625]
Currently the mini FloatingActionButton has a hit target of 40x40. To accommodate a11y users it should be at least 48x48.

**Comment by goderbauer** [comment]
General idea: the GestureDetector should wrap a container of appropriate size (48x48) and the container has the actual smaller FAB as a child.

A challenge might be to limit the ink response to just the FAB.
